util/src/main/java/tc/oc/pgm/util/nms/packets/BlockEntity.java

```diff
@@ -0,0 +1,14 @@
+package tc.oc.pgm.util.nms.packets;
+
+import org.bukkit.Material;
+import org.bukkit.entity.Entity;
+
+public interface BlockEntity {
+    boolean isDisplayEntity();
```

Comment:
Again this breaks encapsulation, the user of this api (the projectile module) should not need to know or care how the underlying magic happens